.github/workflows/ci-check.yml

```diff
@@ -24,7 +24,7 @@ on:
       RUBY_VERSION:
         required: false
         type: string
-        default: "2.7.4"
+        description: Default Ruby version
```

Comment:
A default is no longer being set here. What would the default ruby version be, and will this error out if not set?

Reply:
that's expected the downstream has the default values